librtmp.so.1

after update this morning several issues with librtmp.so.1 not found

ldd /usr/local/bin/curl
        linux-vdso.so.1 (0x00007fffefac6000)
        libcurl.so.4 => /usr/local/lib64/libcurl.so.4 (0x00007cda5e5db000)
        libssl.so.1.0.0 => /usr/local/lib64/libssl.so.1.0.0 (0x00007cda5e369000)
        libcrypto.so.1.0.0 => /usr/local/lib64/libcrypto.so.1.0.0 (0x00007cda5df13000)
        libz.so.1 => /usr/local/lib64/libz.so.1 (0x00007cda5dcf8000)
        libpthread.so.0 => /usr/local/lib64/libpthread.so.0 (0x00007cda5dadb000)
        libc.so.6 => /usr/local/lib64/libc.so.6 (0x00007cda5d730000)
        librtmp.so.1 => not found
        libssh2.so.1 => /usr/local/lib64/libssh2.so.1 (0x00007cda5d504000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007cda5d300000)
        /lib64/ld-linux-x86-64.so.2 (0x00007cda5e852000)

Hi, I still seem to run into the librtmp.so.1 issue when I install `curl` with chromebrew.

I seem to already have a curl, likely from the dev_install command:
```
chronos@localhost ~/Downloads/tools/R-3.4.3 $ curl --version
curl 7.55.1 (x86_64-cros-linux-gnu) libcurl/7.55.1 OpenSSL/1.0.2k zlib/1.2.11
Release-Date: 2017-08-14
Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtsp smtp smtps telnet tftp 
Features: Largefile NTLM SSL libz TLS-SRP UnixSockets HTTPS-proxy 

chronos@localhost ~/Downloads/tools/R-3.4.3 $ which curl
/usr/bin/curl
```

Installing curl with crew next gives me the error:
```
chronos@localhost ~/Downloads/tools/R-3.4.3 $ crew install curl
curl: Command line tool and library for transferring data with URLs.
https://curl.haxx.se/
version 7.57.0
Precompiled binary available, downloading...
Archive downloaded
Unpacking archive, this may take awhile...
Installing...
Performing post-install...
Curl installed!

chronos@localhost ~/Downloads/tools/R-3.4.3 $ which curl
/usr/local/bin/curl

chronos@localhost ~/Downloads/tools/R-3.4.3 $ curl --version
curl: error while loading shared libraries: librtmp.so.1: cannot open shared object file: No such file or directory

chronos@localhost ~/Downloads/tools/R-3.4.3 $ ldd /usr/local/bin/curl
        linux-vdso.so.1 (0x00007fff51a79000)
        libcurl.so.4 => /usr/local/lib64/libcurl.so.4 (0x00007c6162ecb000)
        libssl.so.1.0.0 => /usr/lib64/libssl.so.1.0.0 (0x00007c61632f5000)
        libcrypto.so.1.0.0 => /usr/lib64/libcrypto.so.1.0.0 (0x00007c6162cbe000)
        libz.so.1 => /usr/local/lib64/libz.so.1 (0x00007c6162aa3000)
        libpthread.so.0 => /usr/local/lib64/libpthread.so.0 (0x00007c6162886000)
        libc.so.6 => /usr/local/lib64/libc.so.6 (0x00007c61624db000)
        librtmp.so.1 => not found
        libssh2.so.1 => /usr/local/lib64/libssh2.so.1 (0x00007c61622af000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007c61620ab000)
        /lib64/ld-linux-x86-64.so.2 (0x00007c6163142000)
```

Was trying to install curl with crew too, because another package (R) didn't recognize the right version of libcurl being installed already:
```
configure: error: libcurl >= 7.22.0 library and headers are required with support for https
```

Device is a Pixelbook. I've ran crew update and upgrade.

--- a/packages/curl.rb
+++ b/packages/curl.rb
@@ -3,27 +3,19 @@ require 'package'
 class Curl < Package
   description 'Command line tool and library for transferring data with URLs.'
   homepage 'https://curl.haxx.se/'
-  version '7.57.0'
+  version '7.57.0-1'
   source_url 'https://curl.haxx.se/download/curl-7.57.0.tar.xz'
   source_sha256 'f5f6fd3c72b7b8389969f4fb671ed8532fa9b5bb7a5cae7ca89bc1cea45c7878'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.57.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.57.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.57.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.57.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '73f890553e733f4ad71db78ecc9a80e1e608b13b49faf03ef0a0066ee53bf679',
-     armv7l: '73f890553e733f4ad71db78ecc9a80e1e608b13b49faf03ef0a0066ee53bf679',
-       i686: 'a9a2eaf1560e3480f0d9b2e09f9e5b5a0a92f986d6ed971665d44ff28a299e67',
-     x86_64: 'c9981942058ad8a0d0556df4431e7cfe409f1c37b22e5a1406224ccda7ca5b52',
   })
 
   depends_on 'groff' => :build
   depends_on 'libssh2'
-  depends_on 'openssl' => :build
-  depends_on 'zlibpkg' => :build
+  depends_on 'openssl'
+  depends_on 'rtmpdump'
 
   def self.build
     system './configure',

